lib/api.js

```diff
@@ -0,0 +1,37 @@
+const api = require('@asl/service/api');
+const db = require('@asl/schema');
+const errorHandler = require('@asl/service/lib/error-handler');
+const { NotFoundError } = require('@asl/service/errors');
+const searchRouter = require('./router/search');
+
+module.exports = (settings) => {
+
+  const app = api(settings);
+  const models = db(settings.db);
+
+  app.use((req, res, next) => {
+    req.models = models;
+    next();
+  });
```

Comment:
we probably don't need this if the indexing is going to be triggered via cron job - I was thinking we could have an endpoint that triggers the indexing when we update a licence but that can come later.